.github/workflows/make-gen-delta.yml

```diff
@@ -0,0 +1,46 @@
+name: "make-gen-delta"
+on:
+  - workflow_dispatch
+  - push
+  - workflow_call
+
+permissions:
+  contents: read
+
+jobs:
+  make-gen-delta:
+    name: "Check for uncommitted changes from make gen in extras/kms"
+    runs-on: ${{ fromJSON(vars.RUNNER) }}
```

Comment:
I didn't notice a status posted in the PR status checks.

Looking at the Action logs, it looks like the action didn't get executed because `vars.RUNNER` doesn't exist? https://github.com/hashicorp/go-kms-wrapping/actions/runs/6371729753
> Error when evaluating 'runs-on' for job 'make-gen-delta'. .github/workflows/make-gen-delta.yml (Line: 13, Col: 14): Error parsing fromJson,.github/workflows/make-gen-delta.yml (Line: 13, Col: 14): Error reading JToken from JsonReader. Path '', line 0, position 0.,.github/workflows/make-gen-delta.yml (Line: 13, Col: 14): Unexpected value ''